src/Features/Core/Portable/Microsoft.CodeAnalysis.Features.csproj

```diff
@@ -16,6 +16,7 @@
   </PropertyGroup>
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\..\..\Compilers\Core\Portable\Microsoft.CodeAnalysis.csproj" />
+    <ProjectReference Include="..\..\..\Compilers\CSharp\Portable\Microsoft.CodeAnalysis.CSharp.csproj" />
```

Comment:
nooooooooooooooooooooooooooop

Reply:
this should not be necessary, you can revert this change